### PR TITLE
Add mobile research guidelines

### PR DESCRIPTION
**Pull request by emilywaggoner**
Adding the first draft of mobile research guidelines for the research folder. What do you all think of the Pro/Con tables - they are formatted a little weird, should we keep them?

### CODE REVIEW

File: platform/research/planning/mobile-research-guidelines.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+Ability to see the full movement of the fingers | People without laptops or working web cam can’t use this method
+Relatively easy instruction and setup | The quality of the webcam could be blurry
+
+#### Screensharing method
+The screensharing method requires the participant to download the Zoom app on their phone and share their screen as they complete the tasks of the usability study. 
+
+Pros | Cons
+--------- | ----------
+People without laptops or webcams can use this method | Moderators can not see hand movements
+x | Could be difficult for non-tech savvy people (there might have to be some reconfiguration in the settings section of the phone)
```
Comment (sshein) [inline platform/research/planning/mobile-research-guidelines.md:41]: another "pro" is that Perigean has already helped folks run studies in this way, so it's been socialized with them already. That said, we could work with them on the laptop hugging method

**Review by sshein — APPROVED** [review]
looks good! I like the pros/cons tables. You could also put them as lists if the tables feel like too much to you

**Review by CrystabelReiter — APPROVED** [review]
This looks good! We could add after the line " Invision, UXPin, and staging are viable options for testing mobile flows on VA.gov" that "Note: Participants with assistive tech can only use staging for testing"

**Review by andaleliz — APPROVED** [review]
Looks good! Thanks for putting this together.

**Pull request merged**